README.md

```diff
@@ -298,6 +298,9 @@ resourceInstance.update({
     context: { [field: string]: string };
     [field: string]: any;
   }): Promise<Resource>;
+
+// Links! 🔗🔗🔗🔗
+const incomingLinks: PaginatedList<ResourceLinks> = await resourceInstance.getIncomingLinks({ from: 0, size: 20}: PaginationSettings);
```

Comment:
Are you going to expose a static method too? `Resource.getIncomingLinks`

Reply:
yes!